src/main/kotlin/org/web3j/evm/EmbeddedEthereum.kt

```diff
@@ -139,8 +139,8 @@ class EmbeddedEthereum(
                 @Suppress("UnstableApiUsage")
                 Resources.toString(
                     configuration.genesisFileUrl,
-                    StandardCharsets.UTF_8
-                )
+                    StandardCharsets.UTF_8,
+                ),
```

Comment:
again here, is this a new compilation for kotlin?

Reply:
Seems like it is, as I have upgraded to kotlin 1.9.24 and ktlint to 0.49.1